sdk/logs/src/main/java/io/opentelemetry/sdk/logs/SdkLoggerProviderBuilder.java

```diff
@@ -96,6 +96,24 @@ public SdkLoggerProviderBuilder addLogRecordProcessor(LogRecordProcessor process
     return this;
   }
 
+  /**
+   * Add a log processor as first. {@link LogRecordProcessor#onEmit(Context, ReadWriteLogRecord)}
+   * will be called each time a log is emitted by {@link Logger} instances obtained from the {@link
+   * SdkLoggerProvider}.
+   *
+   * <p>Compared to {@link SdkLoggerProviderBuilder#addLogRecordProcessor(LogRecordProcessor)}, this
+   * method adds the processor to the beginning of the processor pipeline. So the {@code processor}
+   * given will be executed before than the other processors already added.
+   *
+   * @param processor the log processor
+   * @return this
+   */
+  public SdkLoggerProviderBuilder addLogRecordProcessorFirst(LogRecordProcessor processor) {
```

Comment:
This name follows the naming convention of `Deque#addFirst`, albeit "LogRecordProcessor" inserted between "add" and "First". 

While it initially seems like a verbose name, I think its probably the best we can do to balance consistency with the existing "addLogRecordProcessor" and the precedent in java with `Deque#addFirst`.